### PR TITLE
Create example file, update build process and file structure.

### PR DESCRIPTION
**Pull request by andrewrubin**
# Issue
It was difficult to test changes made to this component, so I’d like to add a working example HTML file that can be used alongside the development process. Previously we've had to spin up a test environment with some basic markup, copy + paste the JavaScript, etc.; or we've had to paste our changes into the CodePen example.

# Solution
Add an index HTML file which receives the compiled code and can be served in a development environment.

## Changes
- Changed the bundler from Webpack to **Parcel**. Parcel's ease-of-use and zero-config make this a great choice over Webpack, particularly for a smaller package like the gallery component.
- Added ESLint config.
- Removed all Babel packages, as they're included as dependencies of Parcel.
- Updated readme and JS comments, just because they were both a bit out of date.
- Updated stylesheet to SCSS.
- The NPM scripts for dev and production are updated:
  - `$ yarn start` will create a development build and spin up a live server the the `./dev/` directory.
  - `$ yarn build` will create production-ready versions of the JS and CSS, within the `./dist/` directory.
  - `$ yarn document` will create JSdoc files in the `./docs/` directory.

### CODE REVIEW

**Comment by andrewrubin** [comment]
Hey @marlonmarcello and @liamegan ,
Have a look over these changes when you have some time. Definitely test everything out (see the `package.json` scripts) and let me know what you think can be better!

File: package.json [hunk 1]
```diff
@@ -24,15 +23,15 @@
...
+    "eslint": "^5.16.0",
+    "eslint-config-prettier": "^4.3.0",
+    "eslint-plugin-prettier": "^3.1.0",
+    "jsdoc": "^3.6.3",
+    "prettier": "^1.17.1",
+    "sass": "^1.26.0"
   },
   "dependencies": {
-    "@babel/polyfill": "^7.4.0",
+    "parcel": "^1.12.4",
```
Comment (marlonmarcello) [inline package.json]: Nice

Comment (liamegan) [reply package.json]: Should we move parcel to dev deps?

Comment (andrewrubin) [reply package.json]: Yez. Done!

**Review by liamegan — APPROVED** [review]
I'm good with this.

**Comment by marlonmarcello** [comment]
I am seeing this file as new `src/wtc-gallery-component.scss`
Didn't we have it before?

**Comment by marlonmarcello** [comment]
Also, now that I merged the other PRs could you fix the conflicts please?

**Comment by andrewrubin** [comment]
> I am seeing this file as new `src/wtc-gallery-component.scss`
> Didn't we have it before?

We didn't! We just had a hard-coded `.css` file in the dist folder. Oldschool :)

**Comment by marlonmarcello** [comment]
> > I am seeing this file as new `src/wtc-gallery-component.scss`
> > Didn't we have it before?
> 
> We didn't! We just had a hard-coded `.css` file in the dist folder. Oldschool :)

Awesome, just gotta deal with the conflicts then and we should be good.

**Review by marlonmarcello — APPROVED** [review]

**Comment by andrewrubin** [comment]
Conflicts addressed @marlonmarcello  Thanks!

**Review by marlonmarcello — APPROVED** [review]

**Pull request merged**